applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -327,6 +327,25 @@ static int local_identity_addr_print(void)
 	return 0;
 }
 
+int bt_mgmt_adv_buffer_put(size_t adv_buf_vacant, uint8_t type, size_t data_len,
+			   uint8_t const *data, int *index, struct bt_data *const adv_buf)
```

Comment:
It's a  bit of a rewrite, but I think it would make it easier to read this function if it's in a different order:
```suggestion
int bt_mgmt_adv_buffer_put(struct bt_data *const adv_buf, int *index, size_t adv_buf_vacant, size_t data_len, uint8_t type, uint8_t const *data)
```
Because then it's: 

- Buffer to write to
- The specific index of that buffer
- How much space left there it
- Length
- Type
- Data

Reply:
I see that you changed the order in the last commit. So the old order was close to what I suggest. I remember I had a comment about in vs out in some header file, if that was this order I apologize :P

Reply:
Yes it was, will swap it back!